src/components/date_picker/super_date_picker/quick_select_popover/_quick_select_popover.scss

```diff
@@ -23,9 +23,11 @@
 }
 
 // sass-lint:disable no-important
-.ouiQuickSelectPopover__buttonText {
-  // Override specificity from universal and sibling selectors
-  margin-right: $ouiSizeXS !important;
+.ouiFormControlLayout__prepend{
+  .ouiQuickSelectPopover__buttonText {
+    // Override specificity from universal and sibling selectors
+    margin-right: calc($ouiSizeXS/2) !important;
```

Comment:
why do you need calc when this will resolve to a value?

Also, how did you determine this size?

Reply:
Currently, this is how it is done in rest of OUI rather than using constants. This approach may be helpful  in making things extensible when a theme comes in and changes the `$ouiSizeXS` from 4px to 8px. Other places using this:
https://github.com/search?q=repo%3Aopensearch-project%2Foui+calc%28%24ouiSizeXS+%2F+2%29&type=code

Please let me know if this isn't recommended can hard code to `margin-right: 2px`